assignments/callbacks.js

```diff
@@ -2,29 +2,88 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
+  for(let i = 0; i < arr.length; i++) {
+    if (arr[i] === arr[0]) {
+      cb(arr[0])
+    }
+  }
 }
 
+function logFirstItem (item) {
+  console.log(`${item} is the first item in the array!`);
+}
+
+firstItem(items, logFirstItem)
+
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  cb(arr.length);
+}
+
+function lengthOfArr(len) {
+  console.log(`The length of the array is ${len}!`);
 }
 
+getLength(items, lengthOfArr)
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  cb(arr[arr.length - 1]);
+}
+
+function lastItem(item) {
+  console.log(`${item} is the last item of the array!`);
 }
 
+last(items, lastItem)
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
```

Comment:
Great solution but the function should have passed the sum to the callback. The same for the other 2 problems below it.